homework2/main.py

```diff
@@ -0,0 +1,184 @@
+import requests
+from collections import Counter
+from datetime import datetime, timedelta
+import csv
+import copy
+
+FIELD_NAMES = ['title', 'popularity', 'score', 'last_day_in_cinema']
+API_KEY = "eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiIzMTI3NGFmYTRlNTUyMjRjYzRlN2Q0NmNlMTNkOTZjOSIsInN1YiI6IjVkNmZhMWZmNzdjMDFmMDAxMDU5NzQ4OSIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.lbpgyXlOXwrbY0mUmP-zQpNAMCw_h-oaudAJB6Cn5c8"
+
+
+class MovieDataGrabber:
+    def __init__(self, num_pages=1):
+        self.api_key = API_KEY
+        self.num_pages = num_pages
+        self.url = 'https://api.themoviedb.org/3/'
+        self.data = []
+        self.genres_data = []
```

Comment:
there was a meaning to assign empty list to self.data. you're going to apply extend on that list. but genres_data should be assigned with **None**